.cow.json

```diff
@@ -1,6 +1,6 @@
 {
   "github-slug": "silverstripe/cwp-recipe-cms",
-  "child-stability-inherit": true,
+  "child-stability-inherit": false,
```

Comment:
Change to array with `cwp/cwp` and `cwp/cwp-recipe-core`

Reply:
Cool, done here: https://github.com/silverstripe/cwp-recipe-cms/pull/6